js/iframe-coordinator/libs/Elm.d.ts

```diff
@@ -46,7 +46,21 @@ interface HostProgram {
   };
 }
 
-interface ClientRegistrations {}
+interface ClientRegistrations {
+  [key: string]: ClientConfig;
+}
+
+interface RoutableClientConfig {
+  url: string;
+  assignedRoute: string;
+}
+
+interface BackgroundClientConfig {
+  background: boolean;
+  url: string;
+}
+
+type ClientConfig = RoutableClientConfig | BackgroundClientConfig;
```

Comment:
Here's the union type to allow for configuration of different types.  I can handle this with a more explicit type if we decide that's better.

Reply:
If we only provide a background client config, we will still work ok, right?  If so I think this is fine.